goosebit/updater/controller/v1/routes.py

```diff
@@ -184,11 +188,11 @@ async def download_artifact(_: Request, updater: UpdateManager = Depends(get_upd
     _, software = await updater.get_update()
     if software is None:
         raise HTTPException(404)
-    if software.local:
-        return FileResponse(
-            software.path,
-            media_type="application/octet-stream",
-            filename=software.path.name,
-        )
-    else:
-        return RedirectResponse(url=software.uri)
```

Comment:
I don't think this can be removed, this is for remote file handling, and the polling endpoint always returns this endpoint (it never returns a remote endpoint).

Reply:
The only place we create an URL to this endpoint is here - and this is specific for `software.local`. What am I missing?

```
async def generate_chunk(request: Request, updater: UpdateManager) -> list:
    _, software = await updater.get_update()
    if software is None:
        return []
    if software.local:
        href = str(
            request.url_for(
                "download_artifact",
                dev_id=updater.dev_id,
            )
        )
    else:
        href = software.uri
```

Reply:
I think you're right.